Gale Databases.js

```diff
@@ -1,190 +1,211 @@
 {
-	"translatorID": "e3748cf3-36dc-4816-bf86-95a0b63feb03",
-	"label": "Gale Databases",
-	"creator": "Sebastian Karcher",
-	"target": "^https?://go\\.galegroup\\.com/",
-	"minVersion": "3.0",
-	"maxVersion": "",
-	"priority": 100,
-	"inRepository": true,
-	"translatorType": 4,
-	"browserSupport": "gcsibv",
-	"lastUpdated": "2018-10-07 15:50:14"
+	'translatorID': 'e3748cf3-36dc-4816-bf86-95a0b63feb03',
+	'label': 'Gale Databases',
+	'creator': 'Jim Miazek',
+	'target': '^https?://[^?&]*(?:gale|galegroup|galetesting|ggtest)\.com(?:\:\d+)?/ps/',
+	'minVersion': '3.0',
+	'maxVersion': '',
+	'priority': 100,
+	'inRepository': true,
+	'translatorType': 4,
+	'browserSupport': 'gcsibv',
+	'lastUpdated': '2019-10-11 12:00:00'
 }
 
 /*
 	***** BEGIN LICENSE BLOCK *****
-	
-	Galegroup Translator - Copyright © 2018 Sebastian Karcher 
+
+	Gale Databases Translator - Copyright © 2019
 	This file is part of Zotero.
-	
+
 	Zotero is free software: you can redistribute it and/or modify
 	it under the terms of the GNU Affero General Public License as published by
 	the Free Software Foundation, either version 3 of the License, or
 	(at your option) any later version.
-	
+
 	Zotero is distributed in the hope that it will be useful,
 	but WITHOUT ANY WARRANTY; without even the implied warranty of
 	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 	GNU Affero General Public License for more details.
-	
+
 	You should have received a copy of the GNU Affero General Public License
 	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
-	
+
 	***** END LICENSE BLOCK *****
 */
-function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null}
 
+var GaleZotero = (function () {
+var DATA_TRANSLATOR = '32d59d2d-b65a-4da4-b0a3-bdd3cfb979e7';
```

Comment:
This can just go directly in the `translator.setTranslator()` call below (as in all other translators), with a comment noting that it's RIS.